docs/plugins.md

```diff
@@ -782,6 +782,56 @@ Possible config options:
 
 -   `config` **any** 
 
+## tryTo
+
+Adds global `tryTo` function inside of which all failed steps won't fail a test but will return true/false.
+
+```js
+const result = await tryTo(() => I.see('Welcome'));
```

Comment:
@DavertMik syntax `I.tryTo( () => see("welcome")` would be even a bit nicer :)

Reply:
nope, from JavaScript point of view, it is not :)